core/src/main/res/values/strings.xml

```diff
@@ -120,6 +121,7 @@
   <string name="help_11">ZIM files download in-app are located in the external storage directory in a folder entitled Kiwix.</string>
   <string name="pref_storage">Storage</string>
   <string name="pref_current_folder">Current Folder</string>
+  <string name="pref_history">History</string>
```

Comment:
we already have a string for history so we can just use that, this is not a preference but a title